test/new_fba_tools_tests.pl

```diff
@@ -44,6 +43,16 @@ sub get_ws_name {
         })
    } "build_metabolic_model";
 
+# build_plant_metabolic_model
+lives_ok{
+    $impl->build_plant_metabolic_model({
+	genome_id => "Alyrata_v1.0",
+	genome_workspace => "PlantSEED_v2",
+	fbamodel_output_id =>  "test_model",
+	workspace => get_ws_name()
+				       })
+} "build_plant_metabolic_model";
```

Comment:
Can I hold you to a higher standard than our current codebase? 

It would be **awesome** if you pulled the model back down and did some basic integrity testing for things like number of compounds, reactions, GPRs. This would make the test suite around model building much more robust.

Reply:
I'd thought about that, but I wasn't sure how it works with the way the testing works here in Perl, as each test seems to be restricted to a single function but I can look into it. I even wanted to run FBA and check that there's growth (at least on a plant model).

Reply:
Ok, I won't block on this but am happy to help you figure out better testing soon